promql/functions.go

```diff
@@ -13,6 +13,18 @@
 
 package promql
 
+import (
+	"container/heap"
+	"math"
+	"sort"
+	"strconv"
+	"time"
+
+	clientmodel "github.com/prometheus/client_golang/model"
+
+	"github.com/prometheus/prometheus/storage/metric"
+)
+
 // Function represents a function of the expression language and is
 // used by function nodes.
 type Function struct {
```

Comment:
Will any of this be used outside of the package? If not, unexport the type and its fields.